src/emailUnpackFiles/sendAutoEmail.js

```diff
@@ -0,0 +1,55 @@
+// function to send email to student for low grade
+/*
+data: JSON.stringify(data),
+contentType: 'application/json'
+*/
+import axios from 'axios';
+
+const inputs = {
```

Comment:
Are these inputs for testing?  It might be good to add some comments indicating that they should be deleted before deployment.